src/collections/Breadcrumb/BreadcrumbDivider.js

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+import { customPropTypes } from '../../utils/propUtils'
+import Icon from 'src/elements/Icon/Icon'
+
+/**
+ * A divider sub-component for Breadcrumb component.
+ */
+function BreadcrumbDivider(props) {
+  const {
+    children, icon, className, ...rest,
+  } = props
+
+  const classes = cx(
+    className,
+    'divider',
+  )
+
+  if (icon) {
+    return <Icon className='divider' name={icon} />
+  }
+
+  if (children) {
+    return <div className={classes} {...rest}>{children}</div>
+  }
+
+  return <div className={classes} {...rest}>/</div>
```

Comment:
If we use `defaultProps` for `children`, the `/` will show up in documentation.

```js
  // ...
  
  if (icon) {
    return <Icon className={classes} name={icon} {...rest} />
  }

  return <div className={classes} {...rest}>{children}</div>
}

BreadcrumbDivider.defaultProps = {
  children: '/'
}
```

I've also added `classes` to the icon here so the users classes are still applies in the case of an icon divider.